src/pages/SignUp/SocialLoginLogo/SocialLogo.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import styled from 'styled-components';
+
+function SocialLogo() {
+  return (
+    <SignUpFromWrap>
+      <SignUpTitle>회원가입</SignUpTitle>
+      <SignUpSns>
+        <SignUpSnsTitle>SNS 계정으로 간편하게 회원가입</SignUpSnsTitle>
+        <SnsLogo>
+          <SignUpSnsListItem>
+            <a href="users/auth/facebook">
+              <svg
+                width="48"
+                height="48"
+                viewBox="0 0 48 48"
+                preserveAspectRatio="xMidYMid meet"
+              >
+                <g fill="none" fill-rule="evenodd">
+                  <path
+                    fill="#3B5998"
+                    d="M0 24C0 10.745 10.745 0 24 0s24 10.745 24 24-10.745 24-24 24S0 37.255 0 24z"
+                  ></path>
+                  <path
+                    fill="#FFF"
+                    d="M25.77 35V24h3.384v-3.385h-3.385v-2.538c-.012-.756.08-1.285 1.693-1.692h1.692V13h-3.385c-3.25 0-4.52 1.84-4.23 5.077v2.538H19V24h2.538v11h4.231z"
+                  ></path>
+                </g>
+              </svg>
```

Comment:
이렇게 SVG를 inline으로 넣어야하는 상황에서 svg가 너무 길어서 지저분해보일때는 
이 svg 선언 부분을 파일 최하단등 다른 곳으로 빼버리고
```suggestion
<a href="users/auth/facebook">
  {facebookIconSvg}
</a>
const facebookIconSvg = (
  <svg
    width="48"
    height="48"
    viewBox="0 0 48 48"
    preserveAspectRatio="xMidYMid meet"
  >
    <g fill="none" fill-rule="evenodd">
      <path
        fill="#3B5998"
        d="M0 24C0 10.745 10.745 0 24 0s24 10.745 24 24-10.745 24-24 24S0 37.255 0 24z"
      ></path>
      <path
        fill="#FFF"
        d="M25.77 35V24h3.384v-3.385h-3.385v-2.538c-.012-.756.08-1.285 1.693-1.692h1.692V13h-3.385c-3.25 0-4.52 1.84-4.23 5.077v2.538H19V24h2.538v11h4.231z"
      ></path>
    </g>
  </svg>
);
```
이렇게 하는것도 하나의 방법입니다!

Reply:
오 꿀팁 감사합니다. 연욱님.. 아래와 같이 수정하였습니다. 제가 과연...연욱님같은 경지에 갈수 있을지...ㅠㅠㅠ

```js
          <SignUpSnsListItem>
            <a href="users/auth/facebook">{facebookIconSvg}</a>
            <a href="https://ohou.se/users/auth/kakao">{kakaoIconSvg}</a>
            <a href="users/auth/naver">{naverIconSvg}</a>
          </SignUpSnsListItem>
```